install/first_google_boot.sh

```diff
@@ -215,21 +219,42 @@ function extract_spinnaker_kube_credentials() {
     kube_zone=$MY_ZONE
   fi
 
+  if clear_metadata_to_file "kube_config" $config_path; then
```

Comment:
Not quite, you're enabling Kubernetes, then attempting to write credentials (which can fail due to permissions or gcloud errors). This leaves the spinnaker config in an inconsistent state.